src/services/ReportGenerationService.ts

```diff
@@ -25,7 +25,12 @@ class ReportGenerationService {
             fromDateTime: activity.startTime,
             toDateTime: activity.endTime,
             testStationPNumber: activity.testStationPNumber
-        }).then((testResults: any) => {
+        })
+        .then((testResults) => {
+            // CVSB-7623 - exclude cancelled tests from reports
+            return testResults.filter((test: any) =>test.testStatus !== "cancelled");
```

Comment:
any types can be replaced with type ... can be deferred to phase 2 of Typescript Migration